pkg/kubeclient/signup.go

```diff
@@ -49,3 +50,19 @@ func (c *userSignupClient) Create(obj *crtapi.UserSignup) (*crtapi.UserSignup, e
 	}
 	return result, err
 }
+
+// Update will update an existing UserSignup resource in the cluster, returning an error if something went wrong
+func (c *userSignupClient) Update(obj *crtapi.UserSignup) (*crtapi.UserSignup, error) {
```

Comment:
i think we need to add permissions for `update` to actually work in the deployment?

Reply:
Good catch @tinakurian! We do need to add `update` to https://github.com/codeready-toolchain/registration-service/blob/master/deploy/registration-service.yaml#L27

Reply:
Done

Reply:
@sbryzak friendly reminder - don't forget to also add it in the host-operator `registration-service.yaml`